index.js

```diff
@@ -18,3 +18,5 @@ function responseSchemaCompiler (fjsOpts, { schema /* method, url, httpStatus */
 }
 
 module.exports = SerializerSelector
+module.exports.default = SerializerSelector
+module.exports.SerializerSelector = SerializerSelector
```

Comment:
I thought about this again: How about we also export StandaloneSerializer here?

module.exports.StandaloneSerializer = require('./standalone')

We need then also to export StandaloneSerializer in the typings.

Reply:
I fixed it by exporting one single interface instead of the named export too.

Now, both the interfaces return:

- the function
- the `default` compatibility property

Reply:
Can you please revert the last commit? Now you can not do a named import. 

What I meant whas that you could by 

`module.exports.StandaloneSerializer = require('./standalone')`

do later:

`const { StandaloneSerializer } = require('fast-json-stringify')`

instead of being forced to know the internal structure of the package and being forced to do

`const { StandaloneSerializer } = require('fast-json-stringify/standalone')`

Reply:
Good idea!

Reply:
@Eomm 
Thanks :)

I can not approve again ;)